### ISSUE TITLE
Adding feature to stop/start keylogger after running the client on remote machine

### ISSUE THREAD

**Issue by DragonzMaster** [MaxXor/xRAT#199]
when we open the keylogger form can we send a request to the client to get keylogger statue (On/Off)
if it is turned off a button (will be added to keylogger form ) will enabled and when we press it the keylogger starts its work and vice-versa

**Comment by d3agle** [comment]
There could be a dynamic client settings file that is added that the command handler could read/write to and from

**Comment by MaxXor** [comment]
No, we will just set a lock file in the Logs folder when the Keylogger should be disabled.
We can easily remove/add (on/off) this file.

**Comment by DragonzMaster** [comment]
or it could be a registry value
that depends on file GUID for example

**Comment by yankejustin** [comment]
@MaxXor Like an XML settings file used by the builder form?

**Comment by yankejustin** [comment]
https://github.com/MaxXor/xRAT/blob/eaf89ef97b953d88e5039f221ae866c1b63e1072/Client/Program.cs#L143
We got the handle to the logger, so we should be able to very easily just store it internally and tell it to <code>Dispose</code> when we are done with it!

**Comment by yankejustin** [comment]
After disposing it, we can just poke with this value if we wish to enable or disable on startup https://github.com/MaxXor/xRAT/blob/eaf89ef97b953d88e5039f221ae866c1b63e1072/Client/Program.cs#L138
